sklearn/preprocessing/data.py

```diff
@@ -2028,7 +2028,11 @@ def fit(self, X, y=None):
 
         if y is not None:
             for i in range(len_data):
-                X_key = tuple(X.take(i, axis=0).take(inclusion_used).flatten())
+                X_key_1 = X.take(i, axis=0)
```

Comment:
The error was in the numpy version. 
Scalars as indices were only introduced in numpy 1.8 and scikit-learn is on an older version, so it didn't work out.